src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml

```diff
@@ -95,106 +96,111 @@
+                                            @if (Model.PageContext != null)
+                                            {
+                                                <span class="govuk-caption-xl">
+                                                    @Html.Raw(Model.PageContext)
+                                                </span>
+                                            }
+                                            @if (hasSectionTitle)
+                                            {
+                                                <span class="govuk-caption-xl">@Model.SectionTitle</span>
+                                            }
 
 
-                                        @if (Model.DisplayType == "PagesWithSections" || Model.DisplayType == "OtherPagesInPagesWithSections")
-                                        {
-                                            <h1 class="govuk-heading-l">
-                                                @Html.Raw(Model.BodyText)
-                                            </h1>
-                                           
-                                        }
-                                        else
-                                        {
-                                            <h1 class="govuk-fieldset__heading">
-                                                @Html.Raw(question.Label)
-                                            </h1>
-                                        }
-                                    </legend>
+                                            @if (Model.DisplayType == "PagesWithSections" || Model.DisplayType == "OtherPagesInPagesWithSections")
+                                            {
+                                                <h1 class="govuk-heading-xl">
+                                                    @Html.Raw(Model.BodyText)
+                                                </h1>
+
+                                            }
+                                            else
+                                            {
+                                                <h1 class="govuk-fieldset__heading">
+                                                    @Html.Raw(question.Label)
+                                                </h1>
+                                            }
+                                        </legend>
+
                                         if (!string.IsNullOrWhiteSpace(@Model.BodyText) && Model.DisplayType != "PagesWithSections" && Model.DisplayType != "OtherPagesInPagesWithSections")
                                         {
                                             @Html.Raw(Model.BodyText)
                                         }
                                         else
+                                        {
+                                            @if (Model.DisplayType == "PagesWithSections" || Model.DisplayType == "OtherPagesInPagesWithSections")
```

Comment:
Consider putting DisplayTypes into an enum instead of magic strings

Reply:
And after considering it, make the change. 😁

Reply:
Raised Jira ticket: https://skillsfundingagency.atlassian.net/browse/APR-1565